deepchem/models/lightning/dc_lightning_module.py

```diff
@@ -0,0 +1,99 @@
+import torch
+
+try:
+  import pytorch_lightning as pl  # noqa
+  PYTORCH_LIGHTNING_IMPORT_FAILED = False
+except ImportError:
+  PYTORCH_LIGHTNING_IMPORT_FAILED = True
+
+
+class DCLightningModule(pl.LightningModule):
+  """DeepChem Lightning Module to be used with Lightning trainer.
+
+  Example code
+  >>> import deepchem as dc
+  >>> from deepchem.models import MultitaskClassifier
+  >>> import numpy as np
+  >>> import torch
+  >>> from torch.utils.data import DataLoader
+  >>> from deepchem.models.lightning.dc_lightning_module
+  ...   import DCLightningModule
+  >>> model = MultitaskClassifier(params)
+  >>> valid_dataloader = DataLoader(test_dataset)
```

Comment:
`test_dataset` isn't defined? I think this might break on doctest. Try running doctest locally with

```
python -m doctest dc_lightning_module.py
```